go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/gorilla/mux v1.7.3
 	github.com/kr/pretty v0.1.0 // indirect
+	github.com/mr-tron/base58 v1.1.2
```

Comment:
what is this new import?

Reply:
this must've been added by the IDE without me noticing it.. will fix